crates/solidity/inputs/language/src/definition.rs

```diff
@@ -2424,7 +2428,7 @@ codegen_language_macros::compile!(Language(
                             enabled = Till("0.6.0"),
                             variants = [
                                 EnumVariant(reference = ModifierInvocation),
-                                EnumVariant(reference = OverrideSpecifier),
+                                EnumVariant(reference = ConstantKeyword, enabled = Till("0.5.0")),
```

Comment:
not sure if `ConstantKeyword` here is intended?

Reply:
It's not, I was stacking PRs, sorry. I've rebased and pushed a single fix commit

Reply:
I see. Thanks!
The latest version here removes `OverrideSpecifier` from `UnnamedFunctionAttribute` compeltely. Is that intended? should we add a changeset if so?

Reply:
The unnamed function was enabled only until 0.6.0 and only from that point they introduced the override specifier, so they were never usable together. I think the current changeset of
```
Mark `override` as being a valid attribute only after 0.6.0
```

states that clearly already. WDYT?

Reply:
You are right. Thank you!